### ISSUE TITLE
Regarding the fixed_buffer output 

### ISSUE THREAD

**Issue by ChengFu0118** [cornell-zhang/bnn-fpga#22]
Dear author,

I am trying to use your weight (params) to directly inference your data by using tensorflow. However, I cannot get the same output result from your hardware. I have reshaped the input image into (batch, C, H,W ) and the weight into (K,C,H,W). But your hardware output (fixed_buffer result before batch norm) is different from my software output. Do you have any idea why is that happening?  (I have also considered the zero padding issue).

**Comment by rzhao01** [comment]
Sorry, I somehow didn't get a notification for this issue. So my reply is very late.

The weights directly from the .zip files are reordered before being loaded into the accelerator (see [here](https://github.com/cornell-zhang/bnn-fpga/blob/master/cpp/accel/AccelSchedule.cpp#L237)). It's likely this is causing the difference.

Do you know if the difference is in ordering of the feature maps or are the feature map values completely different?
